### PR TITLE
Kneth/rn71 ndk23

### PR DESCRIPTION
**Pull request by takameyer**
## What, How & Why?

This closes # ??? 

## ☑️ ToDos

* [ ] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 📝 Update `COMPATIBILITY.md`
* [ ] 🚦 Tests
* [ ] 🔀 Executed flexible sync tests locally if modifying flexible sync
* [ ] 📦 Updated internal package version in consuming `package.json`s (if updating internal packages)
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated

### CODE REVIEW

File: scripts/build-android.js [hunk 1]
```diff
@@ -81,7 +78,7 @@ for (const arch of architectures) {
     "-GNinja",
     `-DANDROID_NDK=${ndkPath}`,
     `-DANDROID_ABI=${arch}`,
-    `-DCMAKE_MAKE_PROGRAM=${sdkPath}/cmake/${cmakeVersion}/bin/ninja`,
+    `-DCMAKE_MAKE_PROGRAM=ninja`,
```
Comment (kraenhansen) [inline scripts/build-android.js:81]: Should we have a pre-condition check to verify that the correct version of `ninja` is going to be used?

File: integration-tests/tests/src/tests/sync/sync-as-local.ts [hunk 2]
```diff
@@ -33,7 +33,6 @@ describe.skipIf(environment.missingServer, "Synced Realm as local", () => {
   });
 
   before(async function (this: RealmContext) {
-    this.timeout(5000);
```
Comment (kraenhansen) [inline integration-tests/tests/src/tests/sync/sync-as-local.ts:36]: We should do something else here, to make it stable when running locally then?

Comment (takameyer) [reply integration-tests/tests/src/tests/sync/sync-as-local.ts:36]: I have tried to find a way to set a default, but haven't been successful.  Perhaps we should just set it in the `package.json`

File: contrib/building.md [hunk 3]
```diff
@@ -113,17 +113,16 @@ export ANDROID_EMULATOR_HOME=$HOME/.android
 export ANDROID_AVD_HOME=$HOME/.android/avd
 
 # Add the Android SDK tools to your PATH
-export PATH=$PATH:$ANDROID_SDK_ROOT/tools/bin
+export PATH=$PATH:$ANDROID_SDK_ROOT/to$ols/bin
```
Comment (kraenhansen) [inline contrib/building.md:116]: This looks like a mistake to me.

```suggestion
export PATH=$PATH:$ANDROID_SDK_ROOT/tools/bin
```

File: scripts/build-android.js (hunk 1 above)
Comment (takameyer) [reply scripts/build-android.js:81]: Probably a good idea.  Merging for now, as we have complaints.  Lets look this over afterwards.

File: integration-tests/tests/src/tests/sync/sync-as-local.ts (hunk 2 above)
Comment (takameyer) [reply integration-tests/tests/src/tests/sync/sync-as-local.ts:36]: Let's look this over later

**Pull request merged**